utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -521,6 +521,46 @@ int main(int argc, char **argv)
 
             break;
         }
+        case 34:
+        {
+            testCaseName = "lut";
+
+            Rpp32f *lutBuffer;
+            hipHostMalloc(&lutBuffer, 65536 * sizeof(Rpp32f));
+            hipMemset(lutBuffer, 0, 65536 * sizeof(Rpp32f));
+            Rpp8u *lut8u = reinterpret_cast<Rpp8u *>(lutBuffer);
+            Rpp16f *lut16f = reinterpret_cast<Rpp16f *>(lutBuffer);
+            Rpp32f *lut32f = reinterpret_cast<Rpp32f *>(lutBuffer);
+            Rpp8s *lut8s = reinterpret_cast<Rpp8s *>(lutBuffer);
+            if (inputBitDepth == 0)
```

Comment:
Very inefficient ways of if else .. switch statements, need to revisit

Reply:
We went over this internally, and these buffers are just sample for the user running the unit test. But I felt it was hard to separately initialize 4 buffers. This is what we need:
One (uchar) buffer of length [256] - with values (256 - i)
One (float) buffer of length [256] - with values (256 - i)/255
One (half) buffer of length [256] - with values (256 - i)/255
One (signed char) buffer of length [256] - with values (128 - i)
We can discuss suggestions from your side Rajy